addons/stock/models/stock_location.py

```diff
@@ -136,11 +142,21 @@ def _compute_warehouse_id(self):
                     loc.warehouse_id = view_by_wh[view_location_id]
                     break
 
+    def _compute_child_internal_location_ids(self):
```

Comment:
add depends instead of override create and write: 
`@api.depends('child_ids.usage', 'child_ids.child_internal_location_ids')` should be correct with the recursive parameter in the fields.
What do you think ?

Reply:
done